views/home.njk

```diff
@@ -111,8 +112,8 @@
         <g id="top-slider">
           <path class="st2" d="M50.8,60.4c-4,0-7.3-3.3-7.3-7.3s3.3-7.3,7.3-7.3V60.4z"/>
           <path class="st1" d="M50.8,45.8c4,0,7.3,3.3,7.3,7.3s-3.3,7.3-7.3,7.3V45.8z"/>
-        </g>
         <g id="mid-slider">
+        </g>
```

Comment:
Looks like this accidentally got shifted down a line.

Reply:
Yep, my keyboard shortcuts get messed up when I switch keyboards and I start doing things like this